front/src/app/components/game/game.component.ts

```diff
@@ -304,8 +342,10 @@ export class GameComponent implements AfterViewInit, OnDestroy, OnChanges {
         }
         this.movePaddle(playerName, bindingMap)
       }
+      this.gameboardColors = this.userService.getColorConfig();
```

Comment:
No need to change the arena colors during the game, as the player is currently playing they should not be changing their profile simultaneously (but if they do, changes will be updated upon the next game)